pkg/ddl/schematracker/info_store.go

```diff
@@ -181,3 +181,9 @@ func (i InfoStoreAdaptor) TableByName(schema, table model.CIStr) (t table.Table,
 	}
 	return tables.MockTableFromMeta(tableInfo), nil
 }
+
+// TableInfoByName implements the InfoSchema interface.
+// nolint:unused
```

Comment:
public function will not trigger "unused" linter?

Reply:
I'm not sure, just copied from exists method like `TableByName` ...

Reply:
How about trying to remove it?

Reply:
ok, I'll try to remove it